db/migrate/20220719113627_add_tables.rb

```diff
@@ -4,15 +4,15 @@ class AddTables < ActiveRecord::Migration[7.0]
   def change
     create_table :task_lists do |t|
       t.boolean :hidden, default: false
-      t.string :name, null: false
+      t.string :name
```

Comment:
Could be done in follow up PR but we should have these validations in the model (and tests for them)